upload-chunked-file.js

```diff
@@ -0,0 +1,43 @@
+function chunkFileUpload(file, serverUrl, chunkSize) {
```

Comment:
**Testing**: Consider writing unit tests to validate the chunking logic, verify that all chunks are transmitted accurately, and ensure proper handling of partial uploads.